Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.6
 
-RUN pip install pipenv==8.2.7
+RUN pip install pipenv==11.10.0
```

Comment:
@MIslam92 Should't this be done in separate PR, so we can track it with our commits to master